README.Rmd

```diff
@@ -21,6 +21,8 @@ You can install arcadiathemeR from Github using the `remotes` package. If you do
 remotes::install_github("Arcadia-Science/arcadiathemeR")
 ```
 
+To use the custom fonts you need to download the `TTF` formatted font files and place in the `Users/YOURUSERNAME/Library/Fonts/` directory. Check out the Arcadia Science Brand Assets page in Notion to find these. This should only need to be performed once even if the package is updated over time. These steps and functionality have only been confirmed to work on Mac OS.
```

Comment:
Might want to suggest that users download and double-click on the fonts in order to let FontBook handle the installation on macOS. I think that would be easier for most users to understand.

Reply:
I think it might also be nice to call out what font will be fallen back to (so users know they can still use the package if they don't have the fonts)

Reply:
addressed